front/cypress/e2e/idea_new_page.cy.ts

```diff
@@ -20,73 +20,82 @@ describe('Idea new page for continuous project', () => {
-    cy.get('#e2e-idea-title-input input').type(value);
-    cy.get('#e2e-idea-title-input input').should('contain.value', value);
-    cy.wait(1000);
+  it('shows an error when no title is provided', () => {
+    // Try to go to the next page
+    cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
+    cy.get('#e2e-idea-title-input .e2e-error-message');
+  });
+
+  it('shows an error when no description is provided', () => {
+    const ideaTitle = randomString(9);
+    cy.get('#e2e-idea-title-input input').type(ideaTitle);
+    cy.get('#e2e-idea-title-input input').should('contain.value', ideaTitle);
+
+    // Go to the description page
+    cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
+
     // Try to go to the next page
     cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
     cy.get('#e2e-idea-description-input .e2e-error-message');
   });
 
   it('shows an error when the title is less than 3 characters long', () => {
-    cy.get('#idea-form');
     cy.get('#e2e-idea-title-input input').type(randomString(2));
     // Try to go to the next page
     cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
     cy.get('#e2e-idea-title-input .e2e-error-message');
   });
 
-  it('shows an error when the description is less than 3 characters long', () => {
-    const title = randomString(10);
-    cy.get('#idea-form');
-    cy.get('#e2e-idea-title-input input').type(title);
-    cy.get('#e2e-idea-title-input input').should('contain.value', title);
+  it('shows no error when the description is less than 3 characters long', () => {
+    const ideaTitle = randomString(10);
+    cy.get('#e2e-idea-title-input input').type(ideaTitle);
+    cy.get('#e2e-idea-title-input input').should('contain.value', ideaTitle);
 
+    const ideaContent = randomString(2);
     cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
+    cy.get('#e2e-idea-description-input .ql-editor').type(ideaContent);
+    cy.get('#e2e-idea-description-input .ql-editor').contains(ideaContent);
 
-    // Try to go to the next page
     cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
-    cy.get('#e2e-idea-description-input .e2e-error-message');
+    cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
+    cy.get('[data-cy="e2e-submit-form"]').click();
+    cy.get('[data-cy="e2e-after-submission"]').should('exist').click();
+    cy.contains(ideaTitle).should('exist');
   });
 
-  it.skip('saves correct location point when provided in URL', () => {
+  it('saves correct location point when provided in URL', () => {
     cy.intercept('POST', '**/ideas').as('submitIdea');
 
     const ideaTitle = randomString(40);
     const ideaContent = randomString(60);
-    const geocodedLocation = 'Maria-Louizasquare 47, 1000 Brussel, Belgium';
-    const lat = 50.84682103382404;
-    const long = 4.378963708877564;
+    const geocodedLocation = 'Korenmarkt 11, 9000 Gent, Belgium';
```

Comment:
I had to change this, because in Brussels, it seems the name changes regularly (different mixes of Dutch, French and English). I don't expect this to happen anymore with "Korenmarkt" (popular spot in Gent).